src/api/mocks/handlers/cartHandlers.ts

```diff
@@ -0,0 +1,95 @@
+import { http, HttpResponse } from 'msw';
+import { cartMockDataMSW } from '../cartData';
+import { allProductsMSW } from '../productListMockData';
+
+const baseURL = import.meta.env.VITE_BASE_URL;
+
+export const getCartHandler = http.get(
+  `${baseURL}/cart-items`,
+  ({ request }) => {
+    const url = new URL(request.url);
+    const page = url.searchParams.get('page');
+    const size = url.searchParams.get('size');
+
+    return HttpResponse.json({
+      content: cartMockDataMSW,
+      pageable: {
+        pageNumber: page,
+        pageSize: size,
+        sort: {
+          empty: false,
+          sorted: true,
+          unsorted: false,
+        },
+        offset: 0,
+        paged: true,
+        unpaged: false,
+      },
+      last: true,
+      totalElements: cartMockDataMSW.length,
+      totalPages: 1,
+      size: 50,
+      number: 0,
+      sort: {
+        empty: false,
+        sorted: true,
+        unsorted: false,
+      },
+      first: true,
+      numberOfElements: cartMockDataMSW.length,
+      empty: cartMockDataMSW.length === 0,
+    });
+  }
+);
+
+export const postCartHandler = http.post(
+  `${baseURL}/cart-items`,
+  async ({ request }) => {
+    const { productId, quantity } = (await request.json()) as {
+      productId: number;
+      quantity: number;
+    };
+    const product = allProductsMSW.find(
+      (product) => product.id.toString() === productId.toString()
+    );
+    if (!product)
+      throw new Error('handlers.ts 에서 Product를 찾을 수 없습니다');
```

Comment:
실제로 서버에서 Product를 찾지 못하면 응답으로 404가 올 것 같아! 
throw new Error 보다는 404 response를 보내주는건 어떻게 생각해? 🤔 

```suggestion
    if (!product)
      return new HttpResponse(null, { status: 404 });
```